.github/workflows/scala.yml

```diff
@@ -0,0 +1,35 @@
+name: Scala CI
+
+on:
+  push:
+    branches: '*'
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - uses: coursier/cache-action@v3
+    - name: Set up JDK 11
+      uses: actions/setup-java@v1
+      with:
+        java-version: 11
+    - name: Run tests
+      run: sbt +test
+
+  release:
+    needs: [test]
+    if: 
+      (github.ref == 'refs/heads/master') || startsWith(github.ref, 'refs/tags/')
```

Comment:
This differs from the travis ci expression:

```
if: ((branch = master AND type = push) OR (tag IS present)) AND NOT fork
```

I think that this covers the intent of the travis configuration. Secrets are not made available to forked repositories with github actions.